tools/CustomMSBuild/Versioning.props

```diff
@@ -22,6 +22,8 @@
   <PropertyGroup>
     <VersionFullSemantic>$(VersionMajor).$(VersionMinor).$(VersionBuildNumber)</VersionFullSemantic>
     <VersionFull>$(VersionFullSemantic).$(VersionRevision)</VersionFull>
+	<VersionFullNumber>$(VersionMajor)$(VersionMinor)$(VersionBuildNumber)</VersionFullNumber>

```

Comment:
1) make them alignment
2) the release link for each version is like "https://docs.microsoft.com/en-us/odata/changelog/odatalib-7x#odatalib-781-release"
3) need add them into the build pipeline, otherwise, the nuspec can't get it.
4) queue a nightly using your PR, and verify the generated .nupkg metadata